ext/spl/php_spl.c

```diff
@@ -742,6 +808,11 @@ PHP_RSHUTDOWN_FUNCTION(spl) /* {{{ */
 		zend_string_release_ex(SPL_G(autoload_extensions), 0);
 		SPL_G(autoload_extensions) = NULL;
 	}
+	if (SPL_G(autoload_classmap)) {
+		zend_hash_destroy(SPL_G(autoload_classmap));
+		FREE_HASHTABLE(SPL_G(autoload_classmap));
```

Comment:
Use `zend_array_release(SPL_G(autoload_classmap))` in conjunction with previous changes.